spam/03_spam_data_slicing_tutorial.py

```diff
@@ -218,7 +220,7 @@ def textblob_polarity(x):
 # Most data points with high-polarity sentiments are strong opinions about the video — hence, they are usually relevant to the video, and the corresponding labels are $0$.
 # We might define a slice here for *product and marketing reasons*, it's important to make sure that we don't misclassify very positive comments from good users.
 
-# %%
+# %% {"tags": ["md-exclude-output"]}
```

Comment:
let's put slice_dataframe and head() in separate cells, then just exclude the slice_dataframe one

Reply:
looks like you did this above :+1:

Reply:
sorry missed this. doing now.